tasks/pecl.yml

```diff
@@ -9,7 +9,11 @@
   become: yes
   command: "pecl install {{ item }}"
   register: php_pecl_result
-  changed_when: php_pecl_result.rc == 0
-  failed_when: not php_pecl_result.stdout is search("already installed") or
-               not php_pecl_result.stdout is search("install ok:")
+  changed_when: php_pecl_result.stdout is search("install ok") or
+                php_pecl_result.stderr is search("install ok")
+  failed_when:
+    - not php_pecl_result.stdout is search("already installed")
+    - not php_pecl_result.stdout is search("install ok")
+    - not php_pecl_result.stderr is search("already installed")
+    - not php_pecl_result.stderr is search("install ok")
```

Comment:
Would the Ansible PEAR module handle this for us?

Reply:
It does indeed once you prefix the package name with `pecl/` :-)